palm/plugins/dbt/templates/containerize/Dockerfile.txt

```diff
@@ -4,4 +4,6 @@ WORKDIR /app
 ENV PYTHONPATH=${PYTHONPATH}:${PWD}
 
 RUN ./scripts/entrypoint.sh
+RUN dbt clean && dbt deps
```

Comment:
update template with the things we learned while pairing?

Reply:
I actually don't think this will change in the end... but I'm wondering whether our solution is _really_ suitable for the dbt community at large.

I'm going to no merge this PR until I have a finalized working solution and we have discussed the implications as a team